src/Microsoft.AspNet.Mvc/MvcServiceCollectionExtensions.cs

```diff
@@ -13,7 +13,15 @@ namespace Microsoft.Framework.DependencyInjection
 {
     public static class MvcServiceCollectionExtensions
     {
-        public static IServiceCollection AddMvc(this IServiceCollection services, IConfiguration configuration = null)
+        public static IServiceCollection AddMvc([NotNull]this IServiceCollection services)
+        {
+            ConfigureDefaultServices(services, null);
+            services.TryAdd(MvcServices.GetDefaultServices(null));
+            return services;
+        }
+
+        public static IServiceCollection AddMvc(
+            [NotNull]this IServiceCollection services, [NotNull]IConfiguration configuration)
```

Comment:
Discussed with Yishai. He does not have a strong opinion on this. Will remove `NotNull` and make the above overload call this.